### PR TITLE
Hide tag assignment menu after assignment changes

### LINKED ISSUE #445: Close tag assignment menu immediately after assign/unassign action

### PR DESCRIPTION
**Pull request by tobias-klein**
This will fix #445.

### CODE REVIEW

**Comment by tobias-klein** [comment]
@zhuiks 
This is a rather simple change. The tag assignment menu is hidden after a 500ms delay. Can you check whether this delay feels right from a user perspective? Should we increase it slightly or even make it shorter? If the menu is hidden without a delay then that is too sudden after the assignment/unassignment of a tag. That's why I added it.

File: app/frontend/components/tags/tag_assignment_menu.js [hunk 1]
```diff
@@ -54,6 +54,12 @@ class TagAssignmentMenu {
     }
   }
 
+  hideTagAssignmentMenuAfterDelay() {
+    setTimeout(() => {
+      this.hideTagAssignmentMenu();
+    }, 500);
```
Comment (zhuiks) [inline app/frontend/components/tags/tag_assignment_menu.js:60]: @tobias-klein I think we can slightly increase it to 600ms

**Review by zhuiks — CHANGES_REQUESTED** [review]
We can slightly increase the delay.

File: app/frontend/components/tags/tag_assignment_menu.js (hunk 1 above)
Comment (tobias-klein) [reply app/frontend/components/tags/tag_assignment_menu.js]: Ok!

**Pull request merged**